README.md

```diff
@@ -14,3 +14,33 @@
 ## Development Notes
 
 For misc development details see [Development Notes](https://github.com/TileDB-Inc/TileDB-Java/wiki/Developer-Notes) wiki entry.
+
+##Compatibility
```

Comment:
A missing space between `##` and `Compatibility` causes a render issue

Reply:
Done.